src/query/service/src/interpreters/interpreter_user_udf_drop.rs

```diff
@@ -68,9 +68,11 @@ impl Interpreter for DropUserUDFInterpreter {
             RoleCacheManager::instance().invalidate_cache(&tenant);
         }
 
+        // TODO: if it is appropriate to return an ErrorCode that contains either meta-service error and UdfNotFound error?
+
         UserApiProvider::instance()
             .drop_udf(&tenant, plan.udf.as_str(), plan.if_exists)
-            .await?;
+            .await??;
```

Comment:
typo？

Reply:
No

Reply:
this method returns a nested result: `Result<Result<(), Error>, Error2>`

Reply:
Got